### PR TITLE
Tutorial: mention Python 3 modification on example

### PR DESCRIPTION
**Pull request by weakish**
- `raw_input`
- `Long`

---

 docs/tutorial.rst | 6 ++++++
 1 file changed, 6 insertions(+)

### CODE REVIEW

File: docs/tutorial.rst [hunk 1]
```diff
@@ -151,6 +151,9 @@ Now let's look at the equivalent Hy program:
 
    (simple-conversation)
 
+Note that ``raw_input`` is for Python 2, if you are using Python 3,
```
Comment (kirbyfan64) [inline docs/tutorial.rst]: @hylang/core Come to think of it, maybe Hy should alias `input` to `raw_input` under Python 2, kind of like it does `map`.

Comment (gilch) [inline docs/tutorial.rst]: @kirbyfan64 didn't we fix that already #259?

Comment (kirbyfan64) [inline docs/tutorial.rst]: @gilch Oops. Then this whole change is unneeded.

File: docs/tutorial.rst [hunk 2]
```diff
@@ -213,6 +216,9 @@ supports the Common Lisp method of quoting:
    => '(1 2 3)
    (1L 2L 3L)
 
+Note the output is in Python 2. In Python 3 it is `(1 2 3)`
+because Python 3 does not support `Long`.
```
Comment (kirbyfan64) [inline docs/tutorial.rst]: This is kind of confusing. Maybe, in the earlier sections of the tutorial, it should mention the int <> long difference and how Hy handles it, and then, down here, just refer to that section.

**Comment by gilch** [comment]
I think the docs should use Python3 by default and only mention Python2 as a special case when necessary.

**Comment by Kodiologist** [comment]
@gilch That's not a bad idea, but it's worth its own PR.

At any rate, it seems that consensus is against this PR, at least as it stands.

**Pull request closed without merging**